<launch>

  <param name="/doors_detector_node/parameter_frame" type="string" value="base_footprint" />

  <param name="/doors_detector_node/door_min_height" value="1.1" />
  <param name="/doors_detector_node/door_max_height" value="3.0" />
  <param name="/doors_detector_node/door_min_width" value="0.75" />
  <param name="/doors_detector_node/door_max_width" value="1.2" />

  <param name="/doors_detector_node/handle_min_height" value="0.7" />
  <param name="/doors_detector_node/handle_max_height" value="1.1" />

  <node pkg="door_handle_detector" type="doors_detector_node" name="doors_detector" output="screen" respawn="true">
    <remap from="full_cloud" to="snapshot_cloud" />
  </node>

</launch>
